lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>I can't see the outputs here, but a few percent is not small. It would be nice to find an example where the pass rate changes by less than 1%. Maybe by using 1/10 of the points?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
I tried with 1/10 and got differences of less than 1%

Reply:
<p>Great!</p>